offload/test/offloading/bug64959.c

```diff
@@ -6,6 +6,7 @@
 // UNSUPPORTED: amdgcn-amd-amdhsa
 // UNSUPPORTED: nvptx64-nvidia-cuda
 // UNSUPPORTED: nvptx64-nvidia-cuda-LTO
+// UNSUPPORTED: nvptx64-nvidia-cuda-mpi
```

Comment:
I know it's building off of existing stuff, but this really isn't the way. We need a single feature called `mpi`.

Reply:
I think we could have the `mpi` feature, but in cases where only one or a few of the MPI targets lack support for the test, wouldn’t we run into the same problem?